test/dispatchers/touchDispatcherTests.ts

```diff
@@ -52,6 +52,7 @@ describe("Dispatchers", () => {
         div = TestMethods.generateDiv(DIV_WIDTH, DIV_HEIGHT);
         const component = new Plottable.Component();
         component.anchor(div);
+        component.computeLayout();
```

Comment:
1. This is only to fix phantomjs - when a component i s anchored but not computeLayout-ed, it has `position: absolute` but no width/height/left/top properties. phantom treats this as a 0x0 div and forces all DOM elements, including the transform measurer, to be "inside" that 0x0 box, so getBoundingClientRect() doesn't change and our measuring algorithm fails. But chrome/ie/ff all do the "right" thing here so it doesn't really matter.
1. no one anchors without computeLayout; this is a leftover of Plottable's API but we should eventually just merge these (everyone just uses .renderTo() which does both immediately)

Users of plottable don't have to change anything